content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -156,7 +156,7 @@ the class or PV. If a mount option is invalid, the PV mount fails.
 The `volumeBindingMode` field controls when [volume binding and dynamic
 provisioning](/docs/concepts/storage/persistent-volumes/#provisioning) should occur.
 
-By default, the `Immediate` mode indicates that volume binding and dynamic
+The `Immediate` mode indicates that volume binding and dynamic
```

Comment:
I think it's still good to say that Immediate is the default behavior

Reply:
ok, right without specifying any mode, the default is "Immediate"